src/test/resources/com/znsio/sample/e2e/features/amazon_cart.feature

```diff
@@ -0,0 +1,10 @@
+@amazon
+Feature: Amazon
+
+  @web @amazon @amazon-test
+  Scenario: Add to cart
+    Given I launch amazon application
+    And I search for "Iphone 13" and verify results
```

Comment:
Change
`And I search for "Iphone 13" and verify results`
to
`And I search for "Iphone 13"` --> this should automatically do the verification